pages/task/read/views/components/task-status.jsx

```diff
@@ -14,21 +13,7 @@ const getStatusBadge = (status, model) => {
 
 export default function TaskStatus({ task }) {
   const model = get(task, 'data.model');
-  const latestActivity = task.activityLog.filter(a => a.eventName !== 'assign')[0];
-  let { action, status } = latestActivity;
-
-  if (model === 'project') {
-    const isExtension = isDeadlineExtension(latestActivity);
-
-    if (action === 'update' && isExtension) {
-      status = 'with-inspectorate';
-      action = 'with-inspectorate';
-    }
```

Comment:
Is this handled elsewhere now?

Reply:
It was reading the latest activity to get the current status, but because the latest activity might not be a status change it had to do a lot of work to handle those cases.

Instead it can read the current status of the task from the `status` property and save all the work (unless there's something I've missed). Basically, the whole code block can be replaced by `const status = task.status;`

Reply:
I'm assuming at some point the task status was something other than `with-inspectorate` after an extension was triggered but if it seems fine now it probably is...